tests/functional/utils/chart_certification.py

```diff
@@ -171,6 +171,12 @@ def check_index_yaml(self, base_branch, vendor, chart_name, chart_version, logge
                 f"{chart_name} {chart_version} not added to index")
             return False
 
+        if check_provider_type and self.secrets.vendor_type == 'redhat':
```

Comment:
Can you add a comment on why we are checking this condition? A comment helps when anyone later looks at this condition and wondering why we did this. Also this condition might or might not change so it's beneficial to have a record.

Reply:
Agreed. Added comments explaining this check.

Reply:
Thanks! resolving.